odoo/service/server.py

```diff
@@ -473,8 +474,8 @@ def cron_thread(self, number):
             else:
                 _logger.warning("PG cluster in recovery mode, cron trigger not activated")
             cr.commit()
-
-            while True:
+            alive_time = time.time()
+            while config['limit_time_cron_conn'] <= 0 or (time.time() - alive_time) <= config['limit_time_cron_conn']:
```

Comment:
I know we use `time.time()` everywhere, but this file predates py3. We shall use [`time.monotonic`](http://docs.python.org/3/library/time.html#time.monotonic). The difference between the two is that `time.monotonic` will not be affected if one changes the timezone of the server. Rather small case but still, monotonic is more appropriate than time here :P